src/ArgSortMsg.chpl

```diff
@@ -598,17 +653,26 @@ module ArgSortMsg
                          + (2 * here.maxTaskPar * numLocales * 2**16 * 8));
         
             select (gEnt.dtype) {
-            when (DType.Int64) {
-              var e = toSymEntry(gEnt,int);
-              var iv = argsortDefault(e.a);
-              st.addEntry(ivname, new shared SymEntry(iv));
-            }
-            when (DType.Float64) {
-              var e = toSymEntry(gEnt, real);
-              var iv = argsortDefault(e.a);
-              st.addEntry(ivname, new shared SymEntry(iv));
-            }
-            otherwise {return notImplementedError(pn,gEnt.dtype);}
+                when (DType.Int64) {
+                    var e = toSymEntry(gEnt,int);
+                    var iv = argsortDefault(e.a);
+                    st.addEntry(ivname, new shared SymEntry(iv));
+                }
+                when (DType.Float64) {
+                    var e = toSymEntry(gEnt, real);
+                    var iv = argsortDefault(e.a);
+                    st.addEntry(ivname, new shared SymEntry(iv));
+                }
+                otherwise {
+                    var errorMsg = notImplementedError(pn,gEnt.dtype);
+                    try! writeln(generateErrorContext(
```

Comment:
Can we remove the `try!` from this and similar lines? The parent function `throws`, so it should be fine without it. I would eventually like to remove all `try!` from arkouda to prevent server halts, but in the meantime we should avoid introducing new ones.

Reply:
@reuster986 Excellent point! I removed all try! clauses I added and remove try! clauses for a few of the other try! writeln clauses.

Reply:
excellent, thank you!